services/generic/fake/generic.go

```diff
@@ -0,0 +1,36 @@
+// Package fake implements a fake generic service.
+package fake
+
+import (
+	"context"
+
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/generic"
+)
+
+func init() {
+	resource.RegisterComponent(
```

Comment:
```suggestion
	resource.RegisterService(
```

Reply:
😮‍💨 good catch! Ran an RDK server with the following config to ensure things built correctly:
```json
{
  "services": [
    {
      "name": "g",
      "type": "generic",
      "model": "fake"
    }
  ]
}
```